cmd/osbuild-image-tests/main_test.go

```diff
@@ -130,6 +130,22 @@ func deleteTmpfilesdPathFromImageInfoReport(imageInfoReport interface{}, path st
 	}
 }
 
+// Replace the UUID of the root LVM container with a static one.  We do not
+// control this value and so it is not stable across image builds.
+func replaceLVMUUID(imageInfoReport interface{}, path string) {
+	imageInfoMap := imageInfoReport.(map[string]interface{})
+	partitions, exists := imageInfoMap["partitions"]
+	if exists {
+		for _, partition := partitions.([]interface{}) {
+			partitionInfoMap := partition.(map[string]interface{})
+			if islvm, exists := partitionInfoMap["lvm"]; exists && islvm.(bool) == true {
```

Comment:
I think the correct name for the attribute is `LVM2_member`.

Reply:
(at least it's what I've used in my python version of this)

Reply:
I'm checking the `"lvm"` boolean of the partition
```json
      {
        "bootable": false,
        "fstype": "LVM2_member",
        "label": null,
        "lvm": true,
        "lvm.vg": "rootvg",
        "lvm.volumes": {
          "optlv": {
            "fstype": "xfs",
            "label": null,
            "uuid": "6e4ff95f-f662-45ee-a82a-bdf44a2d0b75"
          },
          "rootlv": {
            "fstype": "xfs",
            "label": "root",
            "uuid": "0194fdc2-fa2f-4cc0-81d3-ff12045b73c8"
          }
        },
        "partuuid": "6264D520-3FB9-423F-8AB8-7A0A8E3D3562",
        "size": 10094624256,
        "start": 642777088,
        "type": "E6D6D379-F507-44C2-A23C-238F2A3DF928",
        "uuid": "9UzqmJ-LgKG-Qlid-YeYg-vVnd-Z3dc-V20Ozc"
      },
```

Reply:
Cool that it works the same! It's even neater with this boolean.